multi-currency-account/MCA Sandbox.postman_environment.json

```diff
@@ -1,15 +1,15 @@
 {
-	"id": "ed4c11ff-653f-4f92-abfc-50548c2f73fa",
-	"name": "MCA Sandbox",
+	"id": "19586927-9e4a-4c06-9e89-e8be585f6586",
+	"name": "Wise For Banks - MCA (Sandbox)",
 	"values": [
 		{
 			"key": "client-id",
-			"value": "",
+			"value": "tw-test-bank",
 			"enabled": true
 		},
 		{
 			"key": "client-secret",
-			"value": "",
+			"value": "tw-test-bank",
```

Comment:
We should leave these blank, not good to expose this, the same goes for client-id

Reply:
Done. Thanks!